src/content/development/backports/_index.en.md

```diff
@@ -30,6 +30,7 @@ Backporting a pull request (PR) is automated by running:
 
 `make LOCAL_BUILD=1 backport release=<release-branch> pr=<PR to cherry-pick>`
 
+Please ensure that [shipyard](https://github.com/submariner-io/shipyard) repo is checked out alongside the project.
```

Comment:
```suggestion
Please ensure that [Shipyard's repo](https://github.com/submariner-io/shipyard) is checked out and updated alongside the project (`../<project dir where running make backport>`).
```

Thanks for clarifying @sridhargaddam! Maybe including this example in the docs would help them be more clear too?

Reply:
```suggestion
Since you are running with `LOCAL_BUILD=1`, ensure that [Shipyard's repo](https://github.com/submariner-io/shipyard) is checked out and updated alongside the project (`../<project dir where running make backport>`).
```

And this is the reasoning, correct? Might also be good to include.